<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>网络设置</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>协议类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxNetProtocolType">
          <item>
           <property name="text">
            <string>TCP Server</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TCP Client</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UDP Server</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UDP Client</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>连接地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxNetConnectAddr"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>本地端口号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditNetPort"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>122</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>122</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>122</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>122</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNetConnect">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>621</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>接收缓冲区</string>
    </property>
    <widget class="QTextBrowser" name="textBrowserNetRecv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>601</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>621</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>发送缓冲区</string>
    </property>
    <widget class="QTextEdit" name="textEditNetSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>531</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSend">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>70</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
